common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -186,11 +167,6 @@ else if (shift == 3)
             return new MonetaryFormat().shift(6).minDecimals(0).optionalDecimals(2);
     }
 
-    @Nullable
-    public String getOwnName() {
-        return Strings.emptyToNull(prefs.getString(PREFS_KEY_OWN_NAME, "").trim());
-    }
```

Comment:
This always returns null

Reply:
This was part of a feature that would put the specified name in all QR codes.  Then those who scanned it would see who it came from if their wallet app supported such a feature as displaying labels in QR codes.